Validation/RecoEgamma/python/ElectronMcFakeValidator_gedGsfElectrons_cfi.py

```diff
@@ -66,7 +65,7 @@
 from Configuration.Eras.Modifier_phase2_hgcal_cff import phase2_hgcal
 phase2_hgcal.toModify(
     electronMcFakeValidator,
-    electronCollection = 'ecalDrivenGsfElectronsHGC',
+    electronCollectionEndcaps = 'ecalDrivenGsfElectronsHGC',
     electronCoreCollection = 'ecalDrivenGsfElectronCoresHGC',
     MaxAbsEta = cms.double(3.0),
```

Comment:
Please remove type specification here, as well as inside the `histosCfg` below

Reply:
Please, when you say remove type specification, do you mean :
MaxAbsEta = cms.double(3.0), --> MaxAbsEta = 3.0, 

if yes, only for double or for all variables (int32(50) --> 50, bool(True) --> True, string('') --> ''; ...) ?

Reply:
Yes, and for all variables which were already defined in the original cloned/modified config.
The rationale is explained for example in point 4 of https://twiki.cern.ch/twiki/bin/view/CMS/OpenRecoTasks#Review_and_optimization_of_the_p : it is a safer syntax that may prevent errors/typos in parameter settings

Reply:
By "and for all variables which were already defined in the original cloned/modified config" do you mean all the variables used into the :
from Configuration.Eras.Modifier_phase2_hgcal_cff import phase2_hgcal 
phase2_hgcal.toModify( part 
or even into the electronMcSignalHistosCfg = cms.PSet( part ?
because when I tried to modify the electronMcSignalHistosCfg=cms.PSet part, I have an error :
_TypeError: Nbinxyz does not already exist, so it can only be set to a CMS python configuration type_